.gitmodules

```diff
@@ -13,7 +13,7 @@
 [submodule "lemmy-js-client"]
 	path = lemmy-js-client
 	url = https://github.com/LemmyNet/lemmy-js-client
-	branch = release/v0.19
+	branch = tsoa_1
```

Comment:
This might have to remain a branch for a while, since we shouldn't release this until `v0.20.0`